tests/components/shopping_list/test_init.py

```diff
@@ -18,12 +18,13 @@
     TYPE_RESULT,
 )
 from homeassistant.const import ATTR_NAME
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers import intent
 
 from tests.common import async_capture_events
 
 
-async def test_add_item(hass, sl_setup):
+async def test_add_item(hass: HomeAssistant, sl_setup):
```

Comment:
we should move the typing improvements into an follow up PR

Reply:
These type changes were also applied by another PR